recipe/meta.yaml

```diff
@@ -1,61 +1,157 @@
 {% set name = "plotly" %}
-{% set version = "5.9.0" %}
+{% set version = "5.19.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: b0536e72bbc0b3cf169ac1fd00759d77aae7bb12ae378cdc75c5dc362f5de576
+  sha256: 5ea91a56571292ade3e3bc9bf712eba0b95a1fb0a941375d978cc79432e055f4
 
 build:
   number: 0
-  skip: true # [py<36]
-  script: {{ PYTHON }} -m pip install . --no-deps -vv
+  skip: true  # [py<38]
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
 
 requirements:
   host:
     - python
     - pip
     - setuptools >=40.8.0
     - wheel
+    - jupyterlab >=3.0,<4
```

Comment:
https://github.com/plotly/plotly.py/blob/6745ef1ec7e46b327a06a8c6a0d98e62399d95bd/doc/python/getting-started.md?plain=1#L103

this is also mentioning `ipywidgets` although it should be optional (so maybe good for `run_constrained` section). Not sure if it makes sense to have `plotly` without `ipywidgets` since it's a graphical tool. On the other hand the user of this package may want to use the cmdli version only.

Reply:
@JeanChristopheMorinPerso your take?

Reply:
I might be wrong but I don't think we need `jupyterlab` in the host section. The sdist already contains all the pre-built JS/HTML files that we need I think (https://github.com/plotly/plotly.py/blob/6745ef1ec7e46b327a06a8c6a0d98e62399d95bd/packages/python/plotly/setup.py#L89-L94).

I'm not sure if we should add `run_constrained` though. @cbouss might have an opinion.

Reply:
I got the `jupyterlab` from here 
```
[build-system]
requires = ["jupyterlab~=3.0;python_version>='3.6'", "setuptools>=40.8.0", "wheel"]
```
https://github.com/plotly/plotly.py/blob/6745ef1ec7e46b327a06a8c6a0d98e62399d95bd/packages/python/plotly/pyproject.toml#L2

in their own recipe here: https://github.com/plotly/plotly.py/blob/ed6888614a020619ec9f22c50f647fa7b1e80c05/packages/python/plotly/recipe/meta.yaml#L21C1-L26C1 

they also mention `nodejs`, which should be used to build those `jsdeps` but I'm not sure about what is going on there.

I don't see much in the folder https://github.com/plotly/plotly.py/tree/v5.19.0/packages/python/plotly/jupyterlab_plotly so maybe something gets pre-build using `jupyterlab` and `nodejs`, according to parameters.

either way, if @cbouss agrees on the `run_constrained`, I would add:

```
jupyterlab >=3
ipywidgets >=7.6
```

Reply:
Yeah, like I said, all the JS and HTML files are already pre-compiled in the sdist. You can try to remove it from the host and see what happens. I think it'll work.

Reply:
ok done